docker-compose.yaml

```diff
@@ -28,6 +28,7 @@ services:
       - DATABASE_URL=${DATABASE_URL}
       - DATABASE_USER=${DATABASE_USER}
       - LOG_LEVEL=${LOG_LEVEL}
+      - LOCAL_CONNECTOR_CATALOG_PATH=${LOCAL_CONNECTOR_CATALOG_PATH}
```

Comment:
Do we need to add this to any of the other apps? I think the bootloader is the only place where this is used because that's where we seed things, but just checking in case we use it in the server or elsewhere as well for some reason (i believe the server should just be using whatever is in the db rather than the local provider)

Reply:
I tested around and it seems like its only the boot loader that requires it